src/components/Layout/NavBar.jsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+import '../../styles/navbar.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ]  It's not a good approach to disable linters, fixing linters help us to write a clean, clear, and good codebase, kindly make sure to not disable linters 🙏🏾